pkg/agent/baker_test.go

```diff
@@ -130,7 +130,7 @@ var _ = Describe("Assert generated customData and cseCmd", func() {
 		// CSE
 		cseCommand := baker.GetNodeBootstrappingCmd(config)
 		// Uncomment below line to generate test data in local if agentbaker is changed in generating customData
-		// ioutil.WriteFile(fmt.Sprintf("./testdata/%s/CSECommand", folder), []byte(cseCommand), 0644)
+		ioutil.WriteFile(fmt.Sprintf("./testdata/%s/CSECommand", folder), []byte(cseCommand), 0644)
```

Comment:
Same here. You need to re-comment out the code that writes test data?

Reply:
oops thanks for catching